demos/database table/workflow_database_tables.sql

```diff
@@ -0,0 +1,46 @@
+SET sql_mode = STRICT_ALL_TABLES;
+
+DROP TABLE IF EXISTS Workflow;
+DROP TABLE IF EXISTS Version;
+DROP TABLE IF EXISTS Runs;
+DROP TABLE IF EXISTS Workflow_Data;
+DROP TABLE IF EXISTS Users;
+
+
+CREATE TABLE Workflow(
+	workflow_id INT NOT NULL AUTO_INCREMENT,
+	PRIMARY KEY(workflow_id)
+) ENGINE = InnoDB;
+	
+CREATE TABLE Version(
+	version_num INT NOT NULL, 
+	workflow_id INT NOT NULL,
+	PRIMARY KEY(workflow_id),

```

Comment:
primary key is (workflow_id, version_num)

Reply:
I'll change that.